vts/plugins/scheme-psa-iot/main.go

```diff
@@ -64,7 +64,10 @@ func (s Scheme) GetFormat() proto.AttestationFormat {
 	return proto.AttestationFormat_PSA_IOT
 }
 
-func (s Scheme) SynthKeysFromSwComponent(tenantID string, swComp *proto.Endorsement) ([]string, error) {
+func (s Scheme) SynthKeysFromSwComponent(
```

Comment:
There seems to be no change except the newlines introduced. Any specific reason for this change?

Reply:
To keep the lines short, to prevent wrapping and make them easier to parse.